### PR TITLE
Bump version, add changelog for 0.11.1 final release.

### PR DESCRIPTION
**Pull request by rtibbles**
Bump version, add changelog for 0.11.1 final release.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,6 +3,32 @@ Release Notes
...
+
+- Fix: Fullscreen renderer mode now works in Chrome 71
+- Fix: Account sign up now works when guest access is disabled
+- Fix: Navigating in and out of exercise detail views
+- Fix: Misleading exam submission modal text
+- Fix: Incredibly slow loading when browsing content tree in exam creation
+- Fix: Loading errors for unavailable content in coach reports
+- Fix: Better content import error catching
+- Added: Support for Python 3.7
+- Added: Command to restore availability of content after bad upgrade
```
Comment (indirectlylit) [inline CHANGELOG.rst]: these 'fix' and 'added' prefixes are weird: they seem redundant/contradictory with the 'added'/'fixed' headers

File: CHANGELOG.rst [hunk 2]
```diff
@@ -3,6 +3,32 @@ Release Notes
...
+
+Internationalization and localization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Added Fufulde Mboroore (fv as ff_CM)
```
Comment (indirectlylit) [inline CHANGELOG.rst]: I don't think we should say anything about language codes in the changelog

File: CHANGELOG.rst [hunk 3]
```diff
@@ -3,6 +3,32 @@ Release Notes
...
+- Support for RTL EPubs
+
+Changed or fixed
+~~~~~~~~~~~~~~~~
+
+- Fix: Fullscreen renderer mode now works in Chrome 71
+- Fix: Account sign up now works when guest access is disabled
+- Fix: Navigating in and out of exercise detail views
+- Fix: Misleading exam submission modal text
+- Fix: Incredibly slow loading when browsing content tree in exam creation
```
Comment (radinamatic) [inline CHANGELOG.rst]: :grin:

**Review by radinamatic — APPROVED** [review]
LGTM! :+1:

**Review by indirectlylit — COMMENTED** [review]
recommend:

* make indentation consistent with the rest of the file
* include link to github changelog label like in other releases
* remove reference to language code
* remove prefixes

**Comment by rtibbles** [comment]
Ah, my use of the 'fixed' and ~'added'~ and language codes was from looking at the 0.10.3 changelog which used ~all three of~ those conventions.

**Comment by rtibbles** [comment]
Updated based on @indirectlylit's feedback.

**Comment by indirectlylit** [comment]
10.3 must have slipped by without my review :)

**Comment by radinamatic** [comment]
I must have reviewed that one :blush:

**Pull request merged**